diff --git a/src/tests/services/article.service.test.ts b/src/tests/services/article.service.test.ts
index 4f39562..3ee353f 100644
--- a/src/tests/services/article.service.test.ts
+++ b/src/tests/services/article.service.test.ts
@@ -44,6 +44,7 @@ describe('ArticleService', () => {
         title: 'How to train your dragon',
         description: '',
         body: '',
+        draft: false,
         createdAt: new Date(),
         updatedAt: new Date(),
         authorId: 456,
@@ -105,6 +106,7 @@ describe('ArticleService', () => {
         title: 'How to train your dragon',
         description: '',
         body: '',
+        draft: false,
         createdAt: new Date(),
         updatedAt: new Date(),
         authorId: 456,
